monai/bundle/scripts.py

```diff
@@ -921,6 +962,78 @@ def verify_net_in_out(
     logger.info("data shape of network is verified with no error.")
 
 
+def _export(
+    converter: Callable,
+    parser: ConfigParser,
+    net_id: str,
+    filepath: str,
+    ckpt_file: str,
+    config_file: str,
+    key_in_ckpt: str,
+    update_args: dict,
```

Comment:
I think no need this `update_args` as the passed `parser` should be ready here.

Thanks.

Reply:
Ok, will delete it.